### PR TITLE
Fixed the type definition

### PR DESCRIPTION
**Pull request by linzhaoming**

### CODE REVIEW

**Comment by enj** [comment]
/assign @php-coder

I think the whole line needs to be updated.  Ping me for approval once Slava LGTM changes.

File: pkg/security/securitycontextconstraints/seccomp/types.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 )
 
-// RunAsUserStrategy defines the interface for all uid constraint strategies.
+// SeccompStrategy defines the interface for all uid constraint strategies.
```
Comment (php-coder) [inline pkg/security/securitycontextconstraints/seccomp/types.go]: s/uid/seccomp/

**Comment by php-coder** [comment]
@linzhaoming Please, squash 2 commits into a one.

**Comment by php-coder** [comment]
/lgtm
/thanks :)

@enj You turn.

**Comment by enj** [comment]
Once the 3.7 release branch is cut the bug only merge requirement will be removed and this will become eligible for merge by the bots.

**Comment by linzhaoming** [comment]
Why the PR is not merged for 2 days? @enj

**Comment by enj** [comment]
Read my last comment.

**Pull request merged**